vavr-jcstress/pom.xml

```diff
@@ -0,0 +1,118 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.vavr</groupId>
+        <artifactId>vavr-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>vavr-jcstress</artifactId>
+    <name>Vavr JCStress test sample</name>
+
+    <prerequisites>
+        <maven>3.0</maven>
+    </prerequisites>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>vavr</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openjdk.jcstress</groupId>
+            <artifactId>jcstress-core</artifactId>
+            <version>${jcstress.version}</version>
+        </dependency>
+    </dependencies>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <!--jcstress version to use with this project.-->
+        <jcstress.version>0.1.1</jcstress.version>
+        <!--Name of the test Uber-JAR to generate.-->
+        <uberjar.name>jcstress</uberjar.name>
+    </properties>
+
+    <build>
+        <plugins>
```

Comment:
Please skip the `maven-deploy-plugin` for the `vavr-jcstress` module. See [vavr-benchmark](https://github.com/vavr-io/vavr/blob/3365fd2e5d5d4a31d8ed41579f40db37da11fb8b/vavr-benchmark/pom.xml#L100) for example. I believe that replacing `<artifactId>maven-install-plugin</artifactId>` with `<artifactId>maven-deploy-plugin</artifactId>` should work. 

BTW, can the `jcstress` start in a phase different from `install` (I see your hint `mvn clean install -Pjcstress`)? Then we can skip `maven-install-plugin` too.

Reply:
[Maven lifecycle refernce](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Lifecycle_Reference) ➡️ `verify` is just before install

Reply:
@nfekete ok, thanks, then we can skip both phases

Reply:
@ruslansennov fixed